packages/yew/src/html/classes.rs

```diff
@@ -60,7 +67,7 @@ impl Classes {
         if self.is_empty() {
             *self = classes_to_add
         } else {
-            self.set.extend(classes_to_add.set)
+            Rc::make_mut(&mut self.set).extend(classes_to_add.set.iter().cloned())
```

Comment:
The use of `Rc::make_mut()` is important here as it will avoid cloning the set if it's not necessary